test.js

```diff
@@ -52,6 +53,49 @@ test('Should return 503 on maxEventLoopDelay', t => {
   })
 })
 
+test('Should return 503 on maxEventloopUtilization', t => {
+  const isSupportedVersion = satisfies(valid(coerce(process.version)), '12.19.0 || >=14.0.0')
```

Comment:
you can pass a `skip` option to tap, use that instead

Reply:
Sure will change that :)

Reply:
I have added te `skip` option instead of `end` @mcollina